ResourceStatusSystem/Command/FreeDiskSpaceCommand.py

```diff
@@ -147,11 +147,15 @@ def doMaster(self):
     """
 
     for name in DMSHelpers().getStorageElements():
-      # keeping TB as default
-      diskSpace = self.doNew((name, 'MB'))
-      if not diskSpace['OK']:
-        gLogger.warn("Unable to calculate free/total disk space", "name: %s" % name)
-        gLogger.warn(diskSpace['Message'])
-        continue
+      try:
+        # keeping TB as default
+        diskSpace = self.doNew((name, 'MB'))
+        if not diskSpace['OK']:
+          gLogger.warn("Unable to calculate free/total disk space", "name: %s" % name)
+          gLogger.warn(diskSpace['Message'])
+          continue
+      except Exception as excp:  # pylint: disable=broad-except
+        gLogger.error("Failed to get SE %s FreeDiskSpace information (SE skipped) " % name)
+        gLogger.exception("Operation finished  with exception: ", lException=excp)
```

Comment:
`continue` ?

Reply:
Well, I don't know why someone has put continue at the end of a for loop (within the if). I retained it.

Reply:
Hum, probably code moved with a copy paste :-)